src/main/kotlin/no/nav/syfo/util/MetrikkUtil.kt

```diff
@@ -0,0 +1,17 @@
+package no.nav.syfo.util
+
+import no.nav.syfo.metrics.GEN_COUNTER
+import no.nav.syfo.model.MedisinskVurdering
+
+fun countGenMetrikk(medisinskVurdering: MedisinskVurdering) {
```

Comment:
Kanskje slå sammen ifene noe slikt slik:
`fun countGenMetrikk(medisinskVurdering: MedisinskVurdering) {
    val newDiagnoseCode = listOf("R991", "U071")
    if (medisinskVurdering.hovedDiagnose != null && newDiagnoseCode.contains(medisinskVurdering.hovedDiagnose!!.kode)) {
        GEN_COUNTER.inc()
    } else if (!medisinskVurdering.biDiagnoser.isNullOrEmpty() && medisinskVurdering.biDiagnoser.find { newDiagnoseCode.contains(it.kode) } != null) {
        GEN_COUNTER.inc()
    }
}`

Reply:
Ja, enig, det blir mye bedre! 🙌

Reply:
Likte navnet newDiagnoseCode bedre enn mitt generiske, så endret det også :)

Reply:
hehe:p
Nice, da teller vi disse "nye diagnosene"